doc/cug.tex

```diff
@@ -3972,6 +3964,40 @@ \subsubsection{Trigger Types}
 \lstinline=$CYLC_DIR/examples/satellite/ext-triggers/suite.rc= is a working
 example of a simulated satellite processing suite.
 
+External triggers are not normally needed in date-time cycling suites driven
+by real time data that comes in at regular intervals.  In these cases a data
+retrieval task can be clock-triggered (and have appropriate retry intervals
+supplied) to submit at the expected data arrival time, so little time if any
+is wasted in polling.  For tasks that may have a highly variable arrival
```

Comment:
I think "tasks" here is a typo.  Maybe something like this: "However, if the arrival time of the cycle-point-specific data is highly variable, external triggering can be used with the cycle point embedded in the message:"